src/api/spec/models/package_spec.rb

```diff
@@ -17,6 +17,23 @@
   let(:other_user) { create(:confirmed_user, login: 'other_user') }
   let(:other_user2) { create(:confirmed_user, login: 'other_user2') }
   let(:other_user3) { create(:confirmed_user, login: 'other_user3') }
+  let(:fake_multibuild_results) do
```

Comment:
We don't usually stub... but in this case he has to setup many stuff (interconnect or repos, create a package with a spec, and so on)... Does it worth?